src/services/routePathService.ts

```diff
@@ -0,0 +1,58 @@
+import gql from 'graphql-tag';
+import { ApolloQueryResult } from 'apollo-client';
+import moment from 'moment';
+import apolloClient from '~/util/ApolloClient';
+import { IRoutePath } from '~/models';
+import notificationStore from '~/stores/notificationStore';
+import NotificationType from '~/enums/notificationType';
+import RoutePathFactory from '../factories/routePathFactory';
+
+export default class RoutePathService {
+    public static async fetchRoutePath
+        (routeId: string, startDate: moment.Moment, direction: string): Promise<IRoutePath | null> {
```

Comment:
What means startDate: moment.Moment? Shouldn't this be the real startDate of routePath?

Reply:
moment.Moment is the type for a timestamp when using moment.js